dashboard/app/views/maker/setup.html.haml

```diff
@@ -6,24 +6,74 @@
     max-width: 800px;
   }
 
+  .board-descriptions {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  .circuit-playground-description {
+    width: 45%
+  }
+
+  .microbit-description {
+    width: 45%
+  }
```

Comment:
super nit: these can be combined (unless we know the styles will be diverging soon)
```suggestion
  .circuit-playground-description,
  .microbit-description {
    width: 45%
  }
```